src/wp-includes/option.php

```diff
@@ -2144,20 +2144,47 @@ function update_network_option( $network_id, $option, $value ) {
 	 */
 	$value = apply_filters( "pre_update_site_option_{$option}", $value, $old_value, $option, $network_id );
 
+	$is_multisite = is_multisite();
+
+	/*
+	 * To get the actual raw old value from the database, any existing pre filters need to be temporarily disabled.
+	 * Immediately after getting the raw value, they are reinstated.
+	 * The raw value is only used to determine whether a value is present in the database. It is not used anywhere
+	 * else, and is not passed to any of the hooks either.
+	 */
+	if ( has_filter( "pre_site_option_{$option}" ) ) {
+		global $wp_filter;
+
+		$old_filters = $wp_filter[ "pre_site_option_{$option}" ];
+		unset( $wp_filter[ "pre_site_option_{$option}" ] );
+
+		$raw_old_value                            = get_network_option( $network_id, $option, false );
+		$wp_filter[ "pre_site_option_{$option}" ] = $old_filters;
```

Comment:
The third parameter for `get_network_option()` defaults to `false` so it can be omitted here (and likely above).

Reply:
@joemcgill, I concur with your agreement on this matter. It would be beneficial to include a default value, as is done in some other places. If it's not a blocking issue, we can follow the same.

- [https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/option.php#L1938](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/option.php#L1938)
- [https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/option.php#L2128](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/option.php#L2128)

Your input on this is appreciated.

Reply:
I'm not sure why the other places where this function is called a third parameter is being passed, but after looking at the commit history, I would guess that when the param order for `get_networt_option()` was changed in [r35024](https://core.trac.wordpress.org/changeset/35024), that the default value was swapped and not removed.

Functionally, there is no difference between `get_network_option( $network_id, $option, false );` and `get_network_option( $network_id, $option);` so the third parameter can be safely removed.